### PR TITLE
Made README standard-readme compliant

### PR DESCRIPTION
**Pull request by RichardLitt**
Any issues? How does it look?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,82 +42,109 @@ There's currently a server (https://github.com/haadcode/orbit-server) that track
...
+
+First, clone this repository locally. Then:
+
+```bash
+npm install
+```
+
+### Requirements
+- [Node.js v4.x.x](http://nodejs.org/)
+- [npm](https://npmjs.com)
```
Comment (VictorBjelkholm) [inline README.md]: would be nice to include the version of npm required, as npm@2 and npm@3 works differently

Comment (RichardLitt) [inline README.md]: @haadcode Which is?

Comment (haadcode) [inline README.md]: Both npm@ and npm@3 should work.

**Comment by haadcode** [comment]
General note on this: I'll merge this once I have everything (ie. js-ipfs) merged to master. Given the vacation, and things I need to fix before I can merge, it'll take some time.

**Comment by haadcode** [comment]
@RichardLitt can you rebase your PR against master? I just updated pretty much everything and there might be bunch of conflicts. Also, can we lose the TOC? Not a fan of that...

**Comment by RichardLitt** [comment]
@haadcode I don't want to bikeshed too much in this thread, as it is one of several; can you respond to my post here about the Table of Contents, with your reasonings for wanting to get rid of it? https://github.com/RichardLitt/standard-readme/issues/27

**Comment by haadcode** [comment]
Hey everyone. The README was updated today. Can you rebase this on master, see if it still makes sense and I can review it.

If this PR is obsolete, just close it.

**Pull request closed without merging**